### ISSUE TITLE
Crash in Don't Starve Together

### ISSUE THREAD

**Issue by MatthieuLepers** [squeek502/Better-Console#4]
Hi ! Your mod crashed when i open the console in Don't Starve Together. The message is "Call to undefined method 'SetAllowClipboardPaste' in betterconsolescreen.lua line 250". So, I comment this line in your code, the mod is not crashing anymore but i can't type command in the console, the ENTER_KEY is not working and my typed command is not preccessed.

Can you fix that pls ?

**Comment by squeek502** [comment]
Sorry, I am not active with Don't Starve modding anymore. However, @nsimplex did do some work on the Don't Starve Together version that never got added to the workshop ([forum download](https://forums.kleientertainment.com/files/file/1467-dst-better-console/), [branch](https://github.com/squeek502/Better-Console/tree/together)), so I've added a naive `SetAllowClipboardPaste` crash fix and uploaded that newer version to the workshop.

Let me know if that works for you.
